### ISSUE TITLE
InfluxDB halts(EADDRNOTAVAIL) with a simple load test

### ISSUE THREAD

**Issue by arunoda** [influxdb/influxdb#2065]
This is with influxdb version `0.8.8`. The test was done in OSX. But, I got the similar results on Linux as well.
(I got the same behavior with influxdb `0.9.0-rc14` as well)

This is the load test scripts:

~~~js
var request = require('request');

var sum = 0;

function write() {
  var s = sum;
  var data = [{"name":"kamal.cpu" + sum,"columns":["value"],"points":[[Math.random()]]}];
  request.post('http://localhost:8086/db/dbname/series?u=root&p=root', {
    data: JSON.stringify(data)
  }, function(err, result) {
    if(err) console.log(err);
    sum ++;
    write();
  });
}

write();

setInterval(function() {
  console.log("sum: ", sum/2);
  sum = 0;
}, 2000);
~~~

After couple of seconds, influxdb process stop accepting connections and it does nothing. It's CPU goes to 0 at that time. Then it'll back again after few seconds and keeping looping like this.

I recorded a video. See: http://recordit.co/ukUfZ4SIAl

Is this the expected behavior or is there anything I can do about this. I didn't tune any nobs.

**Comment by pauldix** [comment]
If you're doing a load test, you'll get 50x better performance if you batch up your writes. Without logs, there's nothing we can do to troubleshoot this. We won't be making any updates to the 0.8 line. If you can reproduce this and provide logs and a script against 0.9.0, we'll investigate.

**Comment by arunoda** [comment]
Okay. How to turn on debug logging for 0.9.x version?

**Comment by pauldix** [comment]
There's this flag you can set https://github.com/influxdb/influxdb/blob/master/cmd/influxd/config.go#L110

But it may be too much output. Just the regular log should suffice.

**Comment by arunoda** [comment]
Okay. I did a load test with batching requests. I must say, wow. 

But, the issue I mentioned is still there, if I continuously hit the DB with a lot of requests (with the above code). With batching requests, this error won't arise.

I hope this is due to something with HTTP layer, but not something with the storage. (Just a guess).

I started the test with a clean DB. Here are the logs:https://gist.github.com/arunoda/69bc5d82cfddcf1f7d6d
(But there's no anything unusual)
